app.yaml

```diff
@@ -14,79 +14,54 @@ inbound_services:
 - warmup
 - mail
 
+default_expiration: "10m"
 handlers:
 - url: /favicon.ico
   static_files: assets/favicon.ico
   upload: assets/favicon.ico
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /robots.txt
   static_files: assets/robots.txt
   upload: assets/robots.txt
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /sitemap.xml
   static_files: assets/sitemap.xml
   upload: assets/sitemap.xml
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
 - url: /build
   static_dir: build
   secure: always
-  expiration: 30d
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
+  expiration: "90d"
 - url: /assets
   static_dir: assets
   secure: always
   application_readable: true
-  http_headers:
-    Cache-Control: 'public, max-age=60'
 # Serve js scripts and css files under core/templates/dev/head.
 # This regex allows us to recursively serve js scripts.
 # "\1" inserts text captured by the capture group in the URL pattern.
 - url: /templates/dev/head/(.*\.(js|css))$
   static_files: core/templates/dev/head/\1
   upload: core/templates/dev/head/(.*\.(js|css))$
   secure: always
-- url: /templates/dev/head/(.*\.(html))$
+- url: /templates/dev/head/(.*directive\.html)$
   static_files: core/templates/dev/head/\1
-  upload: core/templates/dev/head/(.*\.(html))$
-  # TODO(vojtechjelinek): Remove application_readable after all
-  # components are referenced directly by URL
-  application_readable: true
+  upload: core/templates/dev/head/(.*directive\.html)$
   secure: always
 - url: /third_party/static
   static_dir: third_party/static
   secure: always
-  http_headers:
-    Cache-Control: 'public, max-age=2592000'
-    Vary: Accept-Encoding
+  expiration: "90d"
 - url: /third_party/generated
   static_dir: third_party/generated
   secure: always
```

Comment:
Just to double-check, this now expires after 10m, is that right?

I worry a bit about stuff here and in the extensions/ sections below -- we don't hash these URL endpoints, correct? Should we actually be removing the no-cache policy for those? It could cause issues when we upgrade a version.

Reply:
We don't use these on production, everything we use is in production is under `/build` or `/third_party/static`.

Reply:
Ah, ok! Then can we add a note to the relevant stanzas saying "the following URLs are only used on dev" and also change them to immediately expire?

Reply:
Now that I think of it it might be good to separate the handlers into sections (prod, dev, misc) and separate them by comments.

Reply:
Yes, that sounds like a good idea. Though I'm not sure what "misc" means, and probably other devs won't either -- maybe there's a better name?

Reply:
I've separated them by production static, develop static and dynamic (these that are handled by our backend code)